pkg/testutils/centralgrpc/connect_to_central.go

```diff
@@ -73,14 +73,19 @@ func UnauthenticatedGRPCConnectionToCentral(t *testing.T) *grpc.ClientConn {
 }
 
 // GRPCConnectionToCentral returns a GRPC connection to Central, which can be used in E2E tests.
-// It fatals the test if there's an error.
-func GRPCConnectionToCentral(t testutils.T) *grpc.ClientConn {
-	return grpcConnectionToCentral(t, func(opts *clientconn.Options) {
-		opts.ConfigureBasicAuth(RoxUsername(t), RoxPassword(t))
-	})
+// It fatals the test if there's an error. If no options provided assumes basic auth is
+// desired.
+func GRPCConnectionToCentral(t testutils.T, optsFuncs ...func(opts *clientconn.Options)) *grpc.ClientConn {
```

Comment:
The extra `optsFuncs` allows for customizing the connection, it allows for authentication with a token (as opposed to basic auth) - which is used when testing limited permissions in this PR.